src/components/admin/RightColAdmin.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <!-- colonne de droite -->
```

Comment:
Not a huge fan of french commentaries, and this one seems a little... useless ? Same at the end of this template.

Reply:
fixed in following commits